jib-core/src/integration-test/java/com/google/cloud/tools/jib/registry/LocalRegistry.java

```diff
@@ -73,7 +74,9 @@ protected void before() throws IOException, InterruptedException {
                   username,
                   password)
               .run();
-      Path tempFolder = Files.createTempDirectory("auth");
+      // Creates the temporary directory in /tmp since that is one of the default directories mounted into Docker.
```

Comment:
Were our tests flaky or broken?

Reply:
Docker for Mac only mounts certain directories like `/tmp` into the VM and the default temporary directory was generated in `/var`, which isn't mounted into the VM and therefore the `htpasswd` did not exist for the integration test container to use (or rather, the folder could not be mounted to `/auth`).